### PR TITLE
Fix range of locn:geometry

### LINKED ISSUE #1293: Range of locn:geometry
You changed (already in DCAT2) the range of locn:geometry from locn:Geometry to rdfs:Literal.
According to the usage note to be "intentionally generic".

But rdfs:Literal is in my opinion more restrictive than locn:Geometry. According to the usage notes and examples of locn:geometry, it allows Literals AND IRIs: https://www.w3.org/ns/locn#locn:geometry

What am I missing?

### PR DESCRIPTION
**Pull request by andrea-perego**
Relevant issue: https://github.com/w3c/dxwg/issues/1293

- Change range from `rdfs:Literal` to `locn:Geometry`
- Revise usage note

Preview: https://raw.githack.com/w3c/dxwg/dcat-fix-issue-1293/dcat/index.html#Property:location_geometry

Diff: https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fw3c.github.io%2Fdxwg%2Fdcat%2F&doc2=https%3A%2F%2Fraw.githack.com%2Fw3c%2Fdxwg%2Fdcat-fix-issue-1293%2Fdcat%2Findex.html#Property:location_geometry

### CODE REVIEW

File: dcat/index.html [hunk 1]
```diff
@@ -2754,20 +2754,20 @@ <h4>Property: geometry</h4>
...
-              <tr><td class="prop">Usage note:</td><td>The range of this property is intentionally generic, with the purpose of allowing different geometry encodings. E.g., the geometry could be encoded as <abbr title="Well-known Text">WKT</abbr> (<a href="http://www.opengis.net/ont/geosparql#wktLiteral"><code title="http://www.opengis.net/ont/geosparql#wktLiteral">geosparql:wktLiteral</code></a> [[GeoSPARQL]]).</td></tr>
+              <tr><td class="prop">Range:</td><td><a data-cite="LOCN#locn:Geometry"><code title="https://www.w3.org/ns/locn#Geometry">locn:Geometry</code></a></td></tr>
+              <tr><td class="prop">Usage note:</td><td>The range of this property (<code title="https://www.w3.org/ns/locn#Geometry">locn:Geometry</code>) allows for any type of geometry specification. E.g., the geometry could be encoded by a literal, as <abbr title="Well-known Text">WKT</abbr> (<a href="http://www.opengis.net/ont/geosparql#wktLiteral"><code title="http://www.opengis.net/ont/geosparql#wktLiteral">geosparql:wktLiteral</code></a> [[GeoSPARQL]]), or represented by a class, as <code>geosparql:Geometry</code> (or any of its subclasses) [[GeoSPARQL]].</td></tr>
```
Comment (dr-shorthair) [inline dcat/index.html:2761]: Indeed, according to the [LOCN RDF implementation](https://www.w3.org/ns/locn.ttl) the range of `locn:geometry` is  `locn:Geometry`. However, `locn:Geometry` is an `rdfs:Class` so I don't think the object of a `locn:geometry` property can be a literal. .

**Comment by riccardoAlbertoni** [comment]
whatever range we decide, we need to update the RDF as well see  PR #1306 .

File: dcat/index.html (hunk 1 above)
Comment (riccardoAlbertoni) [reply dcat/index.html:2761]: Checking the definition of http://www.opengis.net/ont/geosparql#wktLiteral,  wktLiteral is a `rdfs:Datatype`.

[ `rdfs:Datatype`](https://www.w3.org/TR/rdf-schema/#ch_datatype) is defined in RDF 1.1 as the class of datatypes. All instances of `rdfs:Datatype` correspond to the RDF model of a datatype described in the RDF Concepts specification [RDF11-CONCEPTS]. `rdfs:Datatype` is both an instance of and a subclass of `rdfs:Class`. Each instance of `rdfs:Datatype` is a subclass of `rdfs:Literal`.

@dr-shorthair perhaps this helps to explain why the property can have literals?

**Review by agbeltran — APPROVED** [review]
Following the discussion in the meeting where @andrea-perego pointed to the range of the property and usage note [here](https://www.w3.org/ns/locn#locn:geometry), all looks ok to me

File: dcat/index.html (hunk 1 above)
Comment (andrea-perego) [reply dcat/index.html:2761]: @dr-shorthair , @riccardoAlbertoni , actually, `locn:Geometry` is meant to be a kind of an "abstract" class / placeholder for any type of geometry representation and encoding (classes or literals), as said in its usage note (https://www.w3.org/ns/locn#locn:Geometry):

> This class defines the notion of "geometry" at the conceptual level, and it shall be encoded by using different formats (see usage note of the locn:geometry property).

The usage note of `locn:geometry` (https://www.w3.org/ns/locn#locn:geometry) says:

> Depending on how a geometry is encoded, the range of this property may be one of the following:
> - a literal (e.g., WKT - string literal -, GML, KML - XML literal)
> - a geometry class, as those defined in the OGC's GeoSPARQL specification, in the W3C's Basic Geo (WGS84 lat/long) vocabulary, and at schema.org;
> - geocoded URIs, as geo or GeoHash URIs, treated as URI references.

Comment (andrea-perego) [reply dcat/index.html:2761]: @dr-shorthair , @riccardoAlbertoni , can we mark this comment as resolved?

Comment (dr-shorthair) [reply dcat/index.html:2761]: OK - I understand. These mixed types can cause problems with OWL processors though.

**Review by dr-shorthair — APPROVED** [review]

File: dcat/index.html (hunk 1 above)
Comment (riccardoAlbertoni) [reply dcat/index.html:2761]: @andrea-perego I am in favor of this PR. Only I'd suggest merging #1306 in this PR,  to update the RDF  as well. Being the PR that responds to an erratum can be handy to have a unique pull request to point to in the erratum.

Comment (andrea-perego) [reply dcat/index.html:2761]: Just merged PR https://github.com/w3c/dxwg/pull/1306

**Review by riccardoAlbertoni — APPROVED** [review]

**Pull request merged**